products/apache-maven.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Apache maven
+layout: product
+category: build
+versionCommand: mvn --version
+permalink: /maven
+alternate_urls:
+-   /apache-maven
+-   /mvn
+releasePolicyLink: https://maven.apache.org/docs/history.html
+changelogTemplate: "https://github.com/apache/maven/releases/tag/v__LATEST__"
+auto:
+-   git: https://github.com/apache/maven.git
+# See https://rubular.com/r/stJ20etRIblK0J for reference
+    regex: 'https://rubular.com/r/hQMqEiUNHQOGZL'
```

Comment:
```suggestion
# See https://rubular.com/r/LsLqaCD9QrhuIJ
    regex: ^maven-(?<major>\d+)\.(?<minor>\d+)(\.(?<patch>\d+))?$
```

Fixed the regex, but the github repo seems to be missing v1, v2 releases as tags.